gh-131050: Allow CPython test to handle TLS libraries lacking FFDHE ciphersuites (#131051)
python/cpython: Misc/NEWS.d/next/Tests/2025-03-10-18-58-03.gh-issue-131050.FMBAPN.rst

> Some cryptography TLS libraries lack support for "finite field" ephemeral Diffie-Hellman (FFDHE) TLS ciphersuites. This issue proposes modifying ``test_ssl``'s ``test_dh_params`` to skip itself if the underlying TLS library does not support FFDHE.

Reviewer: ```suggestion
``test_ssl.test_dh_params`` is skipped if the underlying TLS library does not support finite-field ephemeral Diffie-Hellman.
```
